csharp/private/macros/setup_basic_nuget_package.bzl

```diff
@@ -1,19 +1,28 @@
 load("//csharp/private:providers.bzl", "CSharpAssembly")
 load("//csharp/private:rules/imports.bzl", "import_library", "import_multiframework_library")
 
+NugetOnlyFrameworks = {
```

Comment:
Should this be restricted to nuget? Should it be a feature of `import_library`?

Reply:
I considered that but it seemed weird to support it in the `target_framework` attr for that rule only, and then you still basically need the hacks because `import_multiframework_library` doesn't support them either... you can put those hacks into `import_multiframework_library` but then it didn't really look much better.

Having the hacks inside the hacky macro that tries to do its best to make a BUILD file kinda makes sense. If you had to do this by hand you'd probably use `import_library` like how this macro is doing it.

Reply:
I think the ends justify the means: it's a hack to use `import_library` like this but it results in correct stuff. I think its still fair to say we can't really support these frameworks in general though.